src/helpers/getBinomialRandomVariableResult.js

```diff
@@ -0,0 +1,15 @@
+// Iterate over sequence_length items, keep each one with probability success_probability
+// @param {Number} sequence_length - Objects
+// @param {Float} success_probability - Probability
+// @return Number of items that were kept
+const getBinomialRandomVariableResult = (
+  sequence_length,
+  success_probability
```

Comment:
We tend to use camelCase in JavaScript, so maybe you could change them for `sequenceLength` and `successProbability`. But otherwise it looks super nice. 👍